.gitignore

```diff
@@ -1,4 +0,0 @@
-*.pyc
-
-/urdf2webots.egg-info
-/dist
```

Comment:
Why removing this file?

Reply:
was done automatically

Reply:
I think it remains just to re-commit this file.